pydantic_ai_slim/pydantic_ai/result.py

```diff
@@ -436,6 +459,25 @@ async def stream_text(self, *, delta: bool = False, debounce_by: float | None =
         else:
             raise ValueError('No stream response or run result provided')  # pragma: no cover
 
+    def stream_text_sync(self, *, delta: bool = False, debounce_by: float | None = 0.1) -> Iterator[str]:
+        """Stream the text result as a sync iterable.
+
+        This is a convenience method that wraps [`self.stream_text`][pydantic_ai.result.StreamedRunResult.stream_text] with `loop.run_until_complete(...)`.
```

Comment:
Let's change all of them like this:

```suggestion
        This is a convenience method that wraps [`stream_text()`][pydantic_ai.result.StreamedRunResult.stream_text] with `loop.run_until_complete(...)`.
```